tests/test.helpers.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const checkIsCorrectlyAligned = (cellsArray, alignment, cursor = null) => {
```

Comment:
I think this function is redundunt
You can check what you need much simpler

Reply:
Described why it is not so simple in the comment before